hwdetect/hwdetect.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+start_log=$(ls /dev)
+
+while true; do
```

Comment:
This script will take 100% load of process. Please implement timeout. In my opinion 1 second is enough.